README.adoc

```diff
@@ -131,20 +127,47 @@ InventoryLivenessCheck.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[]
 ----
 
+A health Check for liveness allows third party services to determine if the application is running. 
+This means that if this procedure fails the application can be discarded (terminated, shutdown).
+The `@Liveness` annotation must be applied on a HealthCheck implementation to define a Liveness 
+check procedure, otherwise, this annotation is ignored.
 This liveness check verifies that the heap memory usage is below 90% of the maximum memory.
 If more than 90% of the maximum memory is used, a status of `DOWN` will be returned. 
 
+The `inventory` microservice should be healthy only when `system` is available. To add this 
+check to the `/health/ready` endpoint, you will create a class that is annotated with the
```

Comment:
To add this  check to the `/health/ready` endpoint, you will create a class that is annotated with the `@Readiness` annotation and implements the `HealthCheck` interface.
--->
To add this  check to the `/health/ready` endpoint, create a class that is annotated with the `@Readiness` annotation and implements the `HealthCheck` interface.